apps/Launcher/syncwidget.cpp

```diff
@@ -225,8 +225,11 @@ void SyncWidget::setSceneFiles(QMap<QString, QString> sceneFiles) {
         const QString& sceneName = keys[i];
 
         QCheckBox* checkbox = new QCheckBox(sceneName);
-        checkbox->setChecked(true);
-
+        checkbox->setChecked(false);
```

Comment:
I'm more in favor of using the default value as it is;  we don't overwrite all the default values elsewhere in the code